app/models/form526_submission.rb

```diff
@@ -336,6 +336,13 @@ def bdd?
     form.dig('form526', 'form526', 'bddQualified') || false
   end
 
+  # Infers that an HTN RRD Claim Processed
+  def rrd_hypertension_claim_processed
+    form_json.include? RapidReadyForDecision::HypertensionUploadManager::HYPERTENSION_PDF_NAME
+  end
```

Comment:
I recommend moving this to `Form526BaseJob` as `self.rrd_claim_processed?`. As much as possible, let's keep RRD-related code separate from the `Form526Submission` ActiveRecord  (to follow separation of concerns practice).

Reply:
Sorry, not sure I'm following understand, @yoomlam — Form526Submission is an ActiveRecord class, is there a base job class I'm not seeing? I can see that Form526Submission has some related records (`SavedClaim::DisabilityCompensation` and `Form526JobStatus`) but these aren't RRD-specific.

Maybe you mean the RapidReadyForDecision processor selector? That would make sense as it gets the `form526_submission`, but I'm not sure if the Sidekiq memory context would be an issue...

Separately, `self.rrd_claim_processed?` would be a class method and wouldn't have access to form submission context, unless there's something about Ruby I'm misunderstanding...

Thanks!

Reply:
`Form526BaseJob` was added yesterday in #9207. You could put move it to `ProcessorSelector` but that's not where I'd first look for a method that checks if the RRD job successfully processed the claim.

I was thinking the method signature would be `self.rrd_claim_processed?(submission)`.